src/lib/input/autosize.ts

```diff
@@ -10,6 +10,9 @@ import {Directive, ElementRef, Input, AfterViewInit} from '@angular/core';
   exportAs: 'mdTextareaAutosize',
   host: {
     '(input)': 'resizeToFitContent()',
+    // Textarea elements that have the directive applied should have a single row by default.
+    // Browsers normally show two rows by default and this makes the minRows and maxRows unusable.
```

Comment:
In theory it could be also for `maxRows` because if someone sets `maxRows=1` it would be still at two rows I think.

But yeah, who would set `maxRows=1` if he wants autosize.